Enterprise-CMCS/macpro-onemac
# OY2 23647: Additional Fields - CPOC Name

Story: https://qmacbis.atlassian.net/browse/OY2-23647
Endpoint: See github-actions bot comment

### Details
Both CMS and State users wish to see the CPOC assigned to a package.  This field is available in SEA Tool as the LEAD_ANALYST_ID with a lookup into the array, LEAD_ANALYST.  We need to surface that data from a SEA Tool event and add it to our package item, build the name from first name, last name, and display the name on each package details page.

### Changes
- updated buildAnyPackage to pull in the LEAD_ANALYST.FIRST_NAME, LEAD_ANALYST.LAST_NAME
- updated the default details for the package details to include the CPOC
- added CPOC as a column in the package list for all users
- included CPOC in the search bar searches (should not actually be a filtered field based on our design)

### Test Plan
1. Log in as a user that can view packages
2. Navigate to dashboard
3. Sort on CPOC column
4. Verify Search bar correctly searches CPOC along with other searchable fields
5. Click on some packages
6. Verify the default value shows if there is no CPOC
7. Verify the correct CPOC name shows, if available
8. Verify the placement, etc matches figma
